entries/one.xml

```diff
@@ -59,6 +59,12 @@ $( "#foo" ).on( "click", function( event ) {
     </code></pre>
     <p>In other words, explicitly calling <code>.off()</code> from within a regularly-bound handler has exactly the same effect.</p>
     <p>If the first argument contains more than one space-separated event types, the event handler is called <em>once for each event type</em>.</p>
+    <pre><code>
+$( "#foo" ).one( "click mouseover", function() {
+  alert( "This could be displayed twice." );
+});
+    </code></pre>
+    <p>In the example above the alert could be displayed twice due to the <i>two</i> event types (<code>click</code> and <code>mouseover</code>).</p>
```

Comment:
Use `em` instead of `i`. They have a [slight different meaning](http://html5doctor.com/i-b-em-strong-element/).